tests/plando/test_plando.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This seems to be a personal test file so should be moved out before merging.

Reply:
It has been deleted